src/apps/vpn/ui/screens/settings/ViewDNSSettings.qml

```diff
@@ -183,9 +211,7 @@ MZViewBase {
                     }
 
                     onEditingFinished: {
-                        if (ipInput.text === "" || VPN.validateUserDNS(ipInput.text)) {
-                            MZSettings.userDNS = ipInput.text
-                        }
+                        maybeSaveChange();
                     }
```

Comment:
Yes. The acceptance criteria does not make too much sense. The only way to see if I wrote an invalid IP is to change view... which triggers the reset of the input.

Reply:
> Saving happens when the view loose visibility (the user changes the screen)

Hmmm, I am getting different results. Heres a video:

https://user-images.githubusercontent.com/15353801/225715779-d47aa226-4164-43ae-b8f8-bb3653f077b4.mov

'2.2.2.2' is clearly getting saved somewhere during the view's lifetime since it remains in the text field after re-init

Reply:
ah yes. This is mac, right? Yeah, `onEditingFinished()` is totally broken on mac. Fixing! Thanks for testing it.

Reply:
Can you test it again?